releases/2.x.md

```diff
@@ -1,3 +1,14 @@
+# Waffle Takeout 2.5
+
+## Waffle Takeout 2.5.0
+_Released 2016.09.26_
+
+
+#### Maintenance Release
```

Comment:
I think we've been starting to follow more of the conventional changelog - similar to https://github.com/btford/grunt-conventional-changelog/blob/master/CHANGELOG.md. Can you please change this title to be one of these types? - https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#type
Perhaps refactor?